jsk_topic_tools/src/lightweight_throttle_nodelet.cpp

```diff
@@ -94,13 +105,21 @@ namespace jsk_topic_tools
       advertised_ = true;
       pub_ = pnh_.advertise(opts);
     }
+
     ros::Time now = ros::Time::now();
-    if ((now - latest_stamp_).toSec() > 1 / update_rate_) {
+
+    if (latest_stamp_ > now) {
+      ROS_WARN("Detected jump back in time.");
```

Comment:
I think it is already enough.
https://github.com/ros/geometry2/commit/619a1329854d8e6fcc78c576a53594f2caadf383